apps/explorer/lib/explorer/celo/events/celo_contract_events/common/transfer_event.ex

```diff
@@ -2,12 +2,40 @@ defmodule Explorer.Celo.ContractEvents.Common.TransferEvent do
   @moduledoc """
   Struct modelling the Transfer event from the Stabletoken, Goldtoken, Erc20, Stabletokenbrl, Stabletokeneur Celo core contracts.
   """
+  alias Explorer.Repo
+  alias Explorer.Chain.{CeloContractEvent, CeloCoreContract}
 
   use Explorer.Celo.ContractEvents.Base,
     name: "Transfer",
     topic: "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"
 
+  alias Explorer.Celo.ContractEvents.EventMap
+
   event_param(:from, :address, :indexed)
   event_param(:to, :address, :indexed)
   event_param(:value, {:uint, 256}, :unindexed)
+
+  def reserve_bolster(epoch_block_number) do
```

Comment:
I'm not sure this is the best place for this functionality, it's only indirectly related to a Transfer and this module isn't the first place I would look to figure out if an epoch block had a bolster transfer. Possibly CeloElectionRewards would be better.

Reply:
moved to `CeloEpochRewards` as it's fetching data specifically connected to epoch rewards